Luciana Godio/style.css

```diff
@@ -0,0 +1,216 @@
+nav{
+    display: flex;
+    align-items: center;
+    background-color: black;
+    width: 100vw;
+    height: 60px;
+    margin: -9px 0 0 -9px;
+    position: sticky;
+    z-index: 1;
+    
+}
+.nav-lista{
+    display: flex;
+    font-weight: bold;
+    font-family:Verdana, Geneva, Tahoma, sans-serif;
+    flex-direction: row;
+    justify-content: flex-end;
+    width: 45vw;
+    list-style-type: none;
+}
+.banner{
+    width: 100px;
+    padding: 20px;
+}
+
+.nav-lista li a{
+    padding-left: 50px;
+    color: #0d680a;
+    text-decoration: none;
+    padding-bottom: 2px;
+}
+
+body{
+    background-color: rgba(1, 2, 1, 0.849);
+    color: rgb(219, 238, 255);
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+h1{
+    
+    font-weight: bold;
+    text-align: center;
+}
+h2{
+    
+    color: rgb(1, 115, 1);
+}
+.i-inicio {
+    margin:auto;
+    display: block;
+}
+
+.centered {
+    width: 500px;
+    height: 880px;
+    /* border: 4px solid green; */
+    text-align: center;
+    margin:auto;
+}
+.container{
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-template-rows: auto auto;
+    gap: 6px 6px;
+    grid-template-areas: 
+    "quote1 quote2"
+    "quote3 .";
+}
+.quote1 {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 0.1fr 1fr;
+    gap: 0px 0px;
+    grid-template-areas:
+            "ldagger"
+            "image-dagger";            
+    grid-area: quote1;
+}
+.ldagger{
+    grid-area: ldagger;
+}
+.image-dagger {
+    grid-area: image-dagger;
+}
+.quote2 {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 0.1fr 1fr;
+    gap: 0px 0px;
```

Comment:
perhaps you can group diplay, grid-template-column, grid-template-rows and gap in another class, and just leave here what is proper for quote2. and the rest create a separated class that can be reused